inference-engine/src/vpu/graph_transformer/src/middleend/passes/adjust_data_layout.cpp

```diff
@@ -359,9 +359,16 @@ Data PassImpl::addConvertedData(
         const Model& model,
         const Data& orig,
         const StridesRequirement& reqs) {
-    auto data = model->duplicateData(
-        orig,
-        "@adjust-strides");
+    auto data = orig;
+
+    if (orig->usage() == DataUsage::Const) {
+        auto newData = model->addNewData(orig->name(), orig->desc());
+        newData->attrs().copyFrom(orig->attrs());
+        data = model->duplicateData(newData, "@adjust-strides");
```

Comment:
Do you add 2 additional data objects here?

Reply:
No, it fixed